python-package/setup.py

```diff
@@ -20,6 +22,22 @@ def read(filename):
     return io.open(os.path.join(CURRENT_DIR, filename), encoding='utf-8').read()
 
 
+def copy_files():
+
+    def copy_files_helper(folder_name):
+        src = os.path.join(CURRENT_DIR, os.path.pardir, folder_name)
+        if os.path.exists(src):
+            dst = os.path.join(CURRENT_DIR, 'include', folder_name)
+            rmtree(dst, ignore_errors=True)
+            copy_tree(src, dst)
+        else:
+            raise Exception('Cannot copy {} folder'.format(src))
+
+    if not os.path.exists(os.path.join(CURRENT_DIR, '_IS_SOURCE_PACKAGE')):
+        copy_files_helper('RGF')
+#        copy_files_helper('FastRGF')
```

Comment:
`        # copy_files_helper('FastRGF')`

Reply:
position of "#" may be strange, I mean.

Reply:
OK, I'll just remove it.